library/Ivoz/Provider/Application/Service/WebPortal/WebThemeFactory.php

```diff
@@ -39,17 +39,27 @@ public function execute(
             default => throw new \RuntimeException('Unknown web portal type'),
         };
 
-        $publicLogoUrl =
-            'https://'
-            . $hostname
-            . '/api/'
-            . $api
-            . '/my/logo/'
-            . (string) $webPortal->getId()
-            . '/'
-            . urlencode(
-                $webPortal->getLogo()->getBaseName() ?? ''
-            );
+        $baseName = $webPortal->getLogo()->getBaseName();
+        if (!$baseName) {
+            $publicLogoUrl =
+                'https://'
+                . $hostname
+                . '/'
+                . $api
+                . '/logo.svg';
+        } else {
+            $publicLogoUrl =
+                'https://'
+                . $hostname
```

Comment:
I would split the same section as 'https://'. $hostname, in variables.

Reply:
done